minio/minio
# Update version of madmin-go to v1.7.3

## Description

It adds the time zone field to the time-info node in the health report.

## Motivation and Context

Capture time zone info in health report so that `health-analyzer` can add check(s) related to the same.

## How to test this PR?

- Register the cluster if not registered using `mc license register {alias}`
- Generate a health report using `mc support diag {alias} --airgap`
- Inspect the contents of the generated report and verify that field `time_zone` is present under `config -> time-info`

## Types of changes

## Checklist:
